src/app/intro/intro.component.tns.html

```diff
@@ -1,15 +1,16 @@
 <ScrollView>
   <Stacklayout>
     <Label text="Examples" style="font-size: 32; margin: 10"></Label>
+    <Button kirby-button text="Action Sheet" [nsRouterLink]="['/examples/action-sheet']" class="btn btn-primary"></Button>
     <Button kirby-button text="Button" [nsRouterLink]="['/examples/button']" class="btn btn-primary"></Button>
     <Button kirby-button text="Floating Action Button" [nsRouterLink]="['/examples/floating-action-button']"
       class="btn btn-primary"></Button>
     <Button kirby-button text="Card" [nsRouterLink]="['/examples/card']" class="btn btn-primary"></Button>
     <Button kirby-button text="Avatar" [nsRouterLink]="['/examples/avatar']" class="btn btn-primary"></Button>
     <Button kirby-button text="Icon" [nsRouterLink]="['/examples/icon']" class="btn btn-primary"></Button>
     <Button kirby-button text="List" [nsRouterLink]="['/examples/list']" class="btn btn-primary"></Button>
-    <Button kirby-button text="Grid" [nsRouterLink]="['/examples/grid']" class="btn btn-primary"></Button>
     <Button kirby-button text="Modal" [nsRouterLink]="['/examples/modal']" class="btn btn-primary"></Button>
+    <Button kirby-button text="Grid" [nsRouterLink]="['/examples/grid']" class="btn btn-primary"></Button>
```

Comment:
alphabetic ordering

Reply:
.. gone wrong .. :D

Reply:
anyway ... I moved them around a bit more so that they fit the same sequence as we have on web. I think that's better than what we had before, but i won't resolve this comment, let me know if you like the new change, otherwise i'll just revert to the way it is on master currently.

Reply:
That's ok, it's just that it's really not related to this PR, so you end up poluting the version history on that line...